main/EDPftvarcon.F90

```diff
@@ -2114,12 +2155,83 @@ subroutine FatesCheckParams(is_master, parteh_mode)
            call endrun(msg=errMsg(sourcefile, __LINE__))
            
         end if
+        
+        ! Check to see if cohort tracking is on if age-senescence is on
+        !----------------------------------------------------------------------------------
+        
+        if ( ( EDPftvarcon_inst%mort_ip_age_senescence(ipft) < check_initialized ) .and. &
+             ( cohort_age_tracking .neqv. .TRUE. ) ) then
+
+           write(fates_log(),*) 'ip_age = ', EDPftvarcon_inst%mort_ip_age_senescence(ipft)
+           write(fates_log(),*) 'cohort_age_tracking = ', cohort_age_tracking
+
+           write(fates_log(),*) 'Age-dependent mortality cannot be on'
+           write(fates_log(),*) 'if cohort age tracking is off'
+           write(fates_log(),*) 'Set cohort_age_tracking to 1'
+           write(fates_log(),*) 'To turn on cohort age tracking or '
+           write(fates_log(),*) 'Set mort_ip_age_senescence to _ '
```

Comment:
Do you really mean _ or is a placeholder for something else?

Reply:
I did mean _. These values are _ by default, and if the user wants to turn size or age dependent mortality on then they need to set these values.